src/Phpro/SoapClient/BridgeBundle/Type/SoapCall.php

```diff
@@ -133,11 +142,16 @@ public function getRequestHeaders()
      */
     public function getResponseBody()
     {
-        $response = new \DOMDocument();
-        $response->preserveWhiteSpace = false;
-        $response->formatOutput = true;
-        $response->loadXML($this->responseBody);
-        return $response->saveXML();
+        if ($this->responseBody) {
```

Comment:
maybe better could be
`try {
$request = new \DOMDocument();
...
return $request->saveXML();
} catch (\Exception $e) {
return $->getMessage();
}`